lib/openshift/console_notification.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ConsoleNotification < ClusterResource
+    RESOURCE = "consolenotification.console.openshift.io"
```

Comment:
my guess is this shall be `consolenotifications` (`s` at the end).

Reply:
```
$ oc get consolenotification.console.openshift.io
No resources found.
```
This does work.

Reply:
@akostadinov 
```
      [08:40:04] INFO> Shell Commands: oc adm policy add-cluster-role-to-user cluster-admin testuser-45 --config=/home/xiaocwan/workdir/localhost-xiaocwan/ocp4_admin.kubeconfig
      clusterrole.rbac.authorization.k8s.io/cluster-admin added: "testuser-45"
      
      [08:40:43] INFO> Exit Status: 0
    Given admin ensures "notification3" consolenotifications.console.openshift.io is deleted after scenario # features/tierN/web/admin-console/customization.feature:241
      Undefined step: "admin ensures "notification3" consolenotifications.console.openshift.io is deleted after scenario" (Cucumber::Undefined)
      features/tierN/web/admin-console/customization.feature:241:in `Given admin ensures "notification3" consolenotifications.console.openshift.io is deleted after scenario'
```

Reply:
@akostadinov I guess there is something wrong with the step definition, `(\w+)` can not match the `consolenotifications.console.openshift.io` the for dot character. 
Suggest to use `(.+)`, it only not to match the white space. Need I update this in here or still use the previous spelling of the resource type? Otherwise it will block our PR or similar scenario.